### ISSUE TITLE
Install Configs

### ISSUE THREAD

**Issue by ax3l** [mpicbg-scicomp/gearshifft#102]
Currently, `make install` only installs the benchmark binaries but not the default configurations. In order to let gearshifft run out-of-the-box when installed (spack, debian package, etc.), one might also need to install the `config` files (e.g. `extents.csv`).

The right place would be `share/gearshifft/` or `etc/gearshifft/` in *nix systems, prepended by the `CMAKE_PREFIX_PATH`.
You might also want to reflect the same structure for simplicity in the source-directory, too.

This will change the default path of the `-f` parameter of your tools. Ideally it is not `../share/gearshifft/extents.csv` relative to the *calling shell's pwd* but `<pathOfBinary>/../share/gearshifft/extents.csv` of the *called* executable that is installed in `CMAKE_PREFIX_PATH`.

**Comment by ax3l** [comment]
update: oh wait, I just saw there seems to be a fallback to some defaults if the file is missing. So you can keep the `-f` syntax but installing example configs might still be a good idea!

**Comment by tdd11235813** [comment]
yeah, as you pointed out there might be some pitfalls which I want to test, so the config files have not been included in the install process yet. Thanks for your support! I want to release a new version next week.

**Comment by ax3l** [comment]
sure, but I just figured out that you already provide a default fallback, so for a first install the binaries alone are already useful :)

**Comment by tdd11235813** [comment]
Pull Request #110 implements cmake installation of the FFT configs. Check out if you have time or wait for the next pre-release.

A propos releasing. Probably it is better to wait for @psteinb to separate the gearshifft source and results into two projects as mentioned in issue #105 . If time permits I implement a small test environment. Afterwards travis would follow.
